api/swagger-spec/oapi-v1.json

```diff
@@ -18514,6 +18514,77 @@
        "$ref": "v1.EnvVar"
       },
       "description": "env contains additional environment variables you want to pass into a builder container"
+     },
+     "triggeredBy": {
+      "type": "array",
+      "items": {
+       "$ref": "v1.BuildTriggerCause"
+      },
+      "description": "TriggeredBy describes which triggers started the most recent update to the buildconfig and contains information about those triggers."
```

Comment:
Also in the godoc, say "build config" or "build configuration" rather than "BuildConfig" or "buildconfig" (make it human readable).